### PR TITLE
Proposal filters using React

### PR DESCRIPTION
**Pull request by beagleknight**
I'll started my adventure with React and the experience is good so far :D.

I have started implementing a bunch of components and playing with props and events. I'm not 100% happy with the current implementation.

The next step is connecting this simple component with our backend.
## Todo
- [x] Fer que els tags siguin de selecció múltiple
- [x] Moure barra a l'esquerra
- [x] Fer que els tags es puguin deseleccionar
- [x] Arreglar ordenació
- [x] Tenir en compte la cerca al filtrat
- [x] Actualitzar tag cloud segons backend response
- [x] Eliminar linies d'accio del filtre si no hi ha eix seleccionat
- [x] Mantenir string de cerca
- [x] Tests

### CODE REVIEW

File: config/initializers/react.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Consul::Application.configure do
+  config.react.addons = true # defaults to false
```
Comment (beagleknight) [inline config/initializers/react.rb:2]: I started using an addon called `cloneWithProps` but it's deprecated favor to `cloneElement` method.

**Comment by josepjaume** [comment]
I've rebased to latest master, @beagleknight, be sure to fetch the source first.

**Comment by beagleknight** [comment]
@josepjaume I'm mostly done with this feature. As a MVP it works very well. Coverwalls is failing because I don't use an old helper method for building the tag cloud. I can try to fix the Coveralls build if you want.

**Comment by beagleknight** [comment]
@josepjaume I think this is the real MVP now. I'm going to veriffy the current test suite and then write a few ones.

**Comment by josepjaume** [comment]
Looks like a :+1: to me!

**Pull request merged**